.env

```diff
@@ -1,8 +1,6 @@
 # This is an example environment file for configuring dynamic values.
```

Comment:
```suggestion
# This is an example environment file for configuring dynamic values.
# This can be removed or changed with better solutions in the future.
# It should *only* be kept as long as there are no sensitive values in here.
```